reference/5.1/Microsoft.PowerShell.Utility/Write-Output.md

```diff
@@ -101,14 +107,14 @@ Accept wildcard characters: False
 
 By default, the `Write-Output` cmdlet always enumerates its output. The **NoEnumerate** parameter
 suppresses the default behavior, and prevents `Write-Output` from enumerating output. The
-**NoEnumerate** parameter has no effect on collections that were created by wrapping commands in
-parentheses, because the parentheses force enumeration.
+**NoEnumerate** parameter has no effect if the command is wrapped in parentheses, because the
+parentheses force enumeration. For example, `(Write-Output 1,2,3)` will still enumerate the array.
 
 > [!IMPORTANT]
-> There is an issue with this switch in Windows PowerShell that is fixed in PowerShell 6.2 and above.
-> When using **NoEnumerate** and explicitly using the **InputObject** parameter, the command will
-> still enumerate. The workaround is to positionally define the **InputObject** parameter and the
-> enumeration will be suppressed.
+> There is an issue with this switch in Windows PowerShell that is fixed in PowerShell 6.2 and
+> above. When using **NoEnumerate** and explicitly using the **InputObject** parameter, the command
+> will still enumerate. The workaround is to positionally define the **InputObject** parameter and
```

Comment:
I know this existed before, but fix the passive voice here, too.

Reply:
While we're at it: "The workaround is to positionally define the **InputObject** parameter" is confusing.

I suggest:
"The workaround is to pass the arguments positionally, without naming `-InputObject`." 

Rationale: no _parameter_ is being _defined_; parameter _values_ or _arguments_ are being _passed_. 

A brief tangent: it's worth getting clarity on the language around these concepts throughout all topics:
* _parameter_ is the value _holder_ that is part of the command's definition, which is _bound_ during argument passing on invocation.
* what is bound to it is either a _parameter value_ or, perhaps better, an _argument_.

Arguments can be _positional_ or _named_ (preceded by their target parameter _name_)

Reply:
I would argue that, that wording makes it more confusing. You are correct that the Parameter is the value holder and in effect is what the function or cmdlet will interact with.  Even positionally defining a parameter, the parameter is what is referenced when used again and not the argument. The parameter is implicitly defined, not "no parameter value is being defined."

Reply:
@chasewilson , there seems to be problem with GitHub's email notifications - I only just saw this now, after our later conversation.

To be clear: I meant that it's important to distinguish between a _parameter_ - which is _defined_, as part of the command's _implementation_  - and its _value_ (its _argument_), which is _passed_ (or _specified_), as part of the command's _invocation_, and then _bound_ to the parameter.

Referring to both the parameter itself and a value situationally bound to it as _parameter_ invites confusion.
Similarly, I  suggest reserving _defining_ for implementations aspect vs. _passing_ or _specifying_ for the invocation aspect.

To put it differently: you're passing _arguments_ (parameter _values_) to a _parameter_, either _positionally_ or _named_ (by explicitly specifying the parameter's name).